charts/datadog/templates/agent-clusterchecks-deployment.yaml

```diff
@@ -156,6 +156,10 @@ spec:
           - name: DD_LOG_LEVEL
             value: {{ .Values.datadog.logLevel | quote }}
           {{- end }}
+          {{- if .Values.datadog.tags }}
+          - name: DD_TAGS
+            value: {{ tpl (.Values.datadog.tags | join " " | quote) . }}
+          {{- end }}
```

Comment:
on which data you don't see the DD_TAGS coming from the clusterchecks?

DD_TAGS should be propagated by the cluster-agent to the cluster checks config.

Reply:
@clamoriniere ,  it's on kubeStateMetricsCore metrics. We are using datadog agent 7.50.3 (on agent, cluster-agent and cluster-check).

On our side, we have set this values:
```
datadog:
  tags: 
    - "perimeter:hprod"
  kubeStateMetricsCore:
    useClusterCheckRunners: true
...
```

On datadog dashboard for metrics:
<img width="1165" alt="image" src="https://github.com/DataDog/helm-charts/assets/5363260/20c73003-31a9-410f-97b8-57f2e7ebaab3">
* on purple the metrics with `DD_TAGS` manually set on clusterchecksrunner
```
clusterChecksRunner:
  envs:
    - name: DD_TAGS
      value: "perimeter:hprod"
```
* on green the metrics without the customs environment variable